app/packages/core/src/plugins/SchemaIO/components/DropdownView.tsx

```diff
@@ -76,7 +76,7 @@ export default function DropdownView(props) {
             Array.isArray(value) && type !== "array"
               ? value.join(separator)
               : value;
-          onChange(path, computedValue);
+          onChange(path, computedValue, schema);
```

Comment:
Also adds a missing schema value here. This is needed for triggering change events for enum properties.